### DISCUSSION TITLE
Hide first page number in Typst

### DISCUSSION THREAD

**Discussion by majdgh92** [typst/typst#2379] [Q&A] [answered]
Is there any way that I can hide the first page number in Typst.

i have tried to use:
`#set page(
  numbering: "1",
)`
but this works when I place it between text but this does not work when I have a content that takes changes dynamically. thanks``

**Comment by 8LWXpg** [comment, marked as answer]
you can use a function to set numbers

```typst
#set page(numbering: (..numbers) => {
  if numbers.pos().at(0) != 1 {
    numbering("1", numbers.pos().at(0))
  }
})
```

**Reply by majdgh92** [reply]
Thank you very much
